README.md

```diff
@@ -94,6 +94,78 @@ Note that it may run on lower end equipment though good performance is not guara
 
 </div>
 
+## Engines
+
+### PyRGG
+
+<table>
+	<tr>
+		<th>Parameter</th>
+		<th>Description</th>
+	</tr>
+	<tr>
+		<td align="center">Vertices Number</td>
+		<td align="center">The total number of vertices in the graph</td>
+	</tr>
+	<tr>
+		<td align="center">Min Edge Number</td>
+		<td align="center">The minimum number of edges connected to each vertex</td>
+	</tr>
+	<tr>
+		<td align="center">Max Edge Number</td>
+		<td align="center">The maximum number of edges connected to each vertex</td>
+	</tr>
+	<tr>
+		<td align="center">Unweighted / Weighted</td>
+		<td align="center">Specifies whether the graph is unweighted or weighted</td>
+	</tr>
+	<tr>
+		<td align="center">Min Weight</td>
+		<td align="center">The minimum weight of the edges (if weighted)</td>
+	</tr>
+	<tr>
+		<td align="center">Max Weight</td>
+		<td align="center">The maximum weight of the edges (if weighted)</td>
+	</tr>
+	<tr>
+		<td align="center">Unsigned / Signed</td>
+		<td align="center">Specifies whether the edge weights are unsigned or signed</td>
+	</tr>
+	<tr>
+		<td align="center">Undirected / Directed</td>
+		<td align="center">Specifies whether the graph is undirected or directed</td>
+	</tr>
+	<tr>
+		<td align="center">No Self Loop / Self Loop</td>
+		<td align="center">Specifies whether self-loops are not allowed or allowed</td>
```

Comment:
how about `wether self-loop is allowed.`?

Reply:
fixed in https://github.com/sepandhaghighi/pyrgg/pull/169/commits/70562bd6151c7afd029f1769076efbd397df781b